openpype/modules/kitsu/kitsu_widgets.py

```diff
@@ -0,0 +1,193 @@
+import os
+
+import gazu
+from Qt import QtWidgets, QtCore, QtGui
+
+from openpype import style
+from openpype.resources import get_resource
+from openpype.settings.lib import (
+    get_local_settings,
+    get_system_settings,
+    save_local_settings,
```

Comment:
Please don't use local settings for module configurations. Like I've mentioned on discord it is not meant for this kind of data and each modification in local settings UI will remove all data that are stored here. Also local settings are stored in mongo database. Please use `OpenPypeSecureRegistry` or other alternative but not this.

Reply:
Sorry if I misunderstood. I'll have a look with `OpenPypeSecureRegistry` but when I tried few days ago I didn't find how to use it.

Reply:
I was tired for sure, I changed it and it seems to be okay, you'll tell me.